web/forms.py

```
@@ -1754,11 +1759,17 @@ class QuizQuestionForm(forms.ModelForm):
         model = QuizQuestion
-        fields = ["text", "question_type", "explanation", "points", "image"]
+        fields = ["text", "question_type", "explanation", "points", "image", "reference_answer"]
         widgets = {
             "text": TailwindTextarea(attrs={"rows": 3, "placeholder": "Question text"}),
             "question_type": TailwindSelect(),
             "explanation": TailwindTextarea(attrs={"rows": 2, "placeholder": "Explanation for the correct answer"}),
+            "reference_answer": TailwindTextarea(
+                attrs={
+                    "rows": 2,
+                    "placeholder": "Reference answer for the questions, important when AI-auto correction",
+                }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)